adapt-and-extend/theming/resourceregistry.rst

```diff
@@ -291,50 +495,175 @@ Resources which are registered into ``portal_javascripts`` or ``portal_css`` reg
+(see ``Products/CMFPlomino/profiles/default/registry.xml`` and for an example of the resources included ``Products/CMFPlomino/browser/static/js/table.js``)
 
-Besides of using the bundle options ``enabled`` and ``expression``, where you can globally or conditionally control the inclusion of bundles, you also have these options:
+.. code-block:: xml
 
-- Controlling via Diazo: Diazo can include or exclude specific bundles, no matter if it's disabled by default.
-  This can be done in the theme's ``manifest.cfg`` file via the options ``enabled-bundles`` and ``disabled-bundles``.
-  Those options get a comma separated list of bundle names (TODO: verify "comma separated list").
+    <records prefix="plone.bundles/plomino"
+             interface='Products.CMFPlone.interfaces.IBundleRegistry'>
+      <value key="resources">
+        <element>plominoformula</element>
+        <element>plominotable</element>
+        <element>plominodesign</element>
+        <element>plominodynamic</element>
+      </value>
+      <value key="enabled">True</value>
+      <value key="depends">plone</value>
+      <value key="jscompilation">++resource++Products.CMFPlomino/js/plomino-compiled.js</value>
+      <value key="csscompilation">++resource++Products.CMFPlomino/css/plomino-compiled.css</value>
+      <value key="last_compilation">2015-12-08 00:00:00</value>
+    </records>
 
-- A browser page can include or exclude a specific bundle by using the API methods from ``Products.CMFPlone.resources``, no matter if it's disabled by default.
+In `Rapido <https://github.com/plomino/rapido.plone>`_, another Plone add-on, the JavaScript registered for the bundle is manually compiled.
+By listing the ``plone`` default bundle as a dependency, this JavaScript is able to rely on Plone default resources such as ``jQuery``, ``mockup`` and the patterns registry being present.
+(see ``rapido/plone/profiles/default/registry.xml`` and ``rapido/plone/browser/rapido.js``)
 
-These are the ``Products.CMFPlone.resources`` API methods:
+.. code-block:: xml
 
-- ``add_bundle_on_request(request, bundle)``: Adds a bundle to the current request by specifying its name.
+    <records prefix="plone.bundles/rapido"
+             interface='Products.CMFPlone.interfaces.IBundleRegistry'>
+      <value key="enabled">True</value>
+      <value key="jscompilation">++resource++rapido.js</value>
+      <value key="csscompilation"></value>
+      <value key="last_compilation">2019-11-26 00:00:00</value>
+      <value key="compile">False</value>
+      <value key="depends">plone</value>
+    </records>
 
-- ``remove_bundle_on_request(request, bundle)``: Removes a bundle to the current request by specifying its name.
+.. _bundle_example_non_amd:
 
-- ``add_resource_on_request(request, bundle)``: Adds an individual resource to the current request by specifying its name.
+Non-AMD Bundles
+***************
```

Comment:
@ebrehault and this code example is derived from [your pointers above](https://github.com/plone/documentation/pull/518/files#r52873699).  Could you read this and verify that it is both true and accurate?